### ISSUE TITLE
Don't Decay the Learning Rate, Increase the Batch Size

### ISSUE THREAD

**Issue by edmondja** [fchollet/keras#8432]
Hello, is anybody going to implement "Don't Decay the Learning Rate, Increase the Batch Size" : https://arxiv.org/abs/1711.00489 from Google in Keras ? If not, is there a way to put custom batch sizes in a single epoch ?

**Comment by nicolewhite** [comment]
instead of using `model.fit`, use `model.train_on_batch` so that you can control the batch size.
